sdks/python/apache_beam/transforms/util.py

```diff
@@ -671,6 +679,60 @@ def WithKeys(pcoll, k):
   return pcoll | Map(lambda v: (k, v))
 
 
+@typehints.with_input_types(typehints.KV[K, V])
+class GroupIntoBatches(PTransform):
+  def __init__(self, batch_size):
+    self.batch_size = batch_size
+
+  @staticmethod
+  def of_size(batch_size):
+    return GroupIntoBatches(batch_size)
+
+  def expand(self, pcoll):
+    input_coder = coders.registry.get_coder(pcoll)
+    if not input_coder.is_kv_coder():
```

Comment:
Do you need to check this? Type hint should ensure that this is of kv type.

Reply:
fixed